cookbook-react/src/recipes/buttons/zecat/styles.module.scss

```
@@ -0,0 +1,41 @@
+$denim: #0B43D1;
+$smalt: #052E91;
+$white: #FFF;
+$black: #000;
+$mercury: #E1E1E1;
```

I's give them more meaningful names here like:
```suggestion
$backroundColor: #0B43D1;
$activeColor: #052E91;
$color: #FFF;
$hoverShadow: 0px 0px 4px 1px rgba($black, 0.2);
$disabledColor: #E1E1E1;
```